exp/addons/internal/controllers/clusterresourcesetbinding_controller.go

```diff
@@ -80,7 +80,12 @@ func (r *ClusterResourceSetBindingReconciler) Reconcile(ctx context.Context, req
 	if err != nil && !apierrors.IsNotFound(err) {
 		return ctrl.Result{}, err
 	}
-
+	// On a backup, the clusterResourceSetBinding will not have an ownerRef set
+	if err == nil {
```

Comment:
Should this be if cluster == nil ?

Reply:
You are right @killianmuldoon. My bad. 
When the ownerRef is not set, `cluster` and `err` would both be `nil`.